landscape.yml

```diff
@@ -5044,6 +5044,13 @@ landscape:
             repo_url: https://github.com/YugaByte/yugabyte-db
             logo: yugabytedb.svg
             crunchbase: https://www.crunchbase.com/organization/yugabyte
+          - item:
+            name: openGemini
+            description: openGemini is an open source distributed time series DBMS with high concurrency, high performance, and high scalability
+            homepage_url: http://www.openGemini.org
+            repo_url: https://github.com/openGemini
```

Comment:
```suggestion
            repo_url: https://github.com/openGemini/openGemini
```

Reply:
I think needs to be a repo not an org because the org is failing the test and the others that pass link to a repo.